views/AnimatedProfile.tsx

```diff
@@ -38,6 +38,10 @@ export class AnimatedProfile extends Component<Props> {
 
   yTranslate = new Animated.Value(0)
 
+  componentWillUnmount() {
```

Comment:
This fixes a bug, that caused the profile not to show if the user followed these steps:
1. Clicked on a user in the "Hangout" list
2. Navigated to another view without closing the profile
3. Went back to "Results" and tried to open a profile, which now would fail

This fix resets the relevant properties in the store, when the component unmounts